### PR TITLE
Aman

### PR DESCRIPTION
**Pull request by AmanKumarSingh01**
Theme data is added in shared preferences, Now one can toggle between light and dark mode form device settings.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -20,9 +20,8 @@ class MyApp extends StatelessWidget {
         child: MaterialApp(
           debugShowCheckedModeBanner: false,
           title: 'Flutter Demo',
-          theme: ThemeData(
-            primarySwatch: Colors.blue,
-          ),
+          theme: AppTheme.lighttheme,
+          darkTheme: AppTheme.darktheme,
```
Comment (yashwp) [inline lib/main.dart:24]: @AmanKumarSingh01 Use camelCase in the naming convention.

File: lib/styles/app_theme.dart [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+  //const values
+  // static const Color _lightprimarycolur = Colors.white;
+
+  static final ThemeData lighttheme = ThemeData(
+    primarySwatch: Colors.blue,
+  );
+
+  static final ThemeData darktheme = ThemeData(
+    brightness: Brightness.dark,
+  );
```
Comment (yashwp) [inline lib/styles/app_theme.dart:15]: @AmanKumarSingh01 How is this is called theming?

Comment (AmanKumarSingh01) [reply lib/styles/app_theme.dart:15]: I saw this on several websites and youtube videos,

Could you be more specific what you want?

Through the file we can manage themes across the app.

https://www.youtube.com/watch?v=RJkiESVJXAk&t=1439s check this youtube video

Comment (AmanKumarSingh01) [reply lib/styles/app_theme.dart:15]: https://techieblossom.com/theming-in-flutter/

File: lib/main.dart (hunk 1 above)
Comment (AmanKumarSingh01) [reply lib/main.dart:24]: camelCase done

**Pull request closed without merging**